plugin/auth_ldap/tests/mtr/auth_ldap_simple.test

```diff
@@ -0,0 +1,44 @@
+
+# This testcase requires the following ldap structure:
+# https://github.com/rroemhild/docker-test-openldap
+# The easiest way to run it is to use the provided docker image
+
```

Comment:
use `count_sessions.inc/wait_until_count_sessions.inc` for MTR test cases establising new connections